onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -5,6 +5,7 @@
 Support for registering ONNX Runtime's built-in contrib ops with
 PyTorch-ONNX exporter (torch.onnx.export).
 """
+from __future__ import annotations
```

Comment:
Please add a comment explaining why this is needed.

Reply:
Done. This is needed for enabling postponed evaluation of annotations so that we don't need to put quotes around torch._C.Value etc.

Reply:
Why is postponed evaluation required in this file?
Is it so that you can do the conditional import?
If so, why do we need the conditional import?

Reply:
postponed evaluation is needed because we are doing the conditional import.

When `TYPE_CHECKING` is true (set by the type checkers like mypy), the module is imported and `def gelu(g, self: torch._C.Value, approximate: str = "none")` can be analyzed.

When `TYPE_CHECKING` is false:
If we don't do `from __future__ import annotations`, Python (<3.11) will complain that `torch._C.Value` is not known. Alternatively, we can put quotes around the type like so `def gelu(g, self: "torch._C.Value", approximate: str = "none")`

`from __future__ import annotations` effectively (as I understand it) adds the quotes for us so we don't have to.

Reply:
Why not remove the future import and remove the `if` around the import? It just seems so much simpler.

Reply:
I can do that too! Sometimes it is necessary to avoid circular imports. It's also the thought of don't import it if it is not used (it is not used in run time / when not type checking). What do you think? I am happy to make the change

Reply:
If it doesn't create a circular import, please just import it unconditionally.

Reply:
Done